File: Iniciante/1048 - Aumento de Salário/C#/1048 - Aumento de Salário.cs
using System;
using System.Globalization; // formata os valores para en-us (.)

class URI {
    static void Main() {
        // variável auxiliar
        double reajuste = 0;
        // entrada do salário inicial
        double sal_ini = double.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);

        // verifica o percentual de reajuste
        if(sal_ini > 0 && sal_ini <= 400) {
            reajuste = 0.15;

        } else if(sal_ini > 400 && sal_ini <= 800) {
            reajuste = 0.12;

        } else if(sal_ini > 800 && sal_ini <= 1200) {
            reajuste = 0.10;

        } else if(sal_ini > 1200 && sal_ini <= 2000) {
            reajuste = 0.07;

        } else if(sal_ini > 2000) {
            reajuste = 0.04;

        }

        // cálculo do salário + reajuste
        double sal_final = sal_ini + (sal_ini*reajuste);

        // saída
        Console.WriteLine("Novo salario: " + sal_final.ToString("0.00", System.Globalization.CultureInfo.InvariantCulture));
        Console.WriteLine("Reajuste ganho: " + (sal_ini*reajuste).ToString("0.00", System.Globalization.CultureInfo.InvariantCulture));
        Console.WriteLine("Em percentual: {0} %", reajuste*100);
        

    }
}

File: Iniciante/1048 - Aumento de Salário/Python/1048 - Aumento de Salário.py
# entrada
sal_ini = float(input())

# identifica o percentual do reajuste
if sal_ini > 0 and sal_ini <= 400:
    reajuste = 0.15

elif sal_ini > 400 and sal_ini <= 800:
    reajuste = 0.12

elif sal_ini > 800 and sal_ini <= 1200:
    reajuste = 0.10

elif sal_ini > 1200 and sal_ini <= 2000:
    reajuste = 0.07

elif sal_ini > 2000:
    reajuste = 0.04

# cálculo do salário + reajuste
sal_final = sal_ini + (sal_ini*reajuste)

# saída
print("Novo salario: {:.2f}".format(sal_final))
print("Reajuste ganho: {:.2f}".format(sal_ini*reajuste))
print("Em percentual:", int(reajuste*100), "%")